app/controllers/candidate_interface/apply_controller.rb

```diff
@@ -0,0 +1,8 @@
+module CandidateInterface
+  class ApplyController < CandidateInterfaceController
+    def show
+      @provider_code = params[:providerCode]
+      @course_code = params[:courseCode]
```

Comment:
I'm assuming the camelcase is already in use on Find?

Reply:
It's not already in use, this is the first time we are defining these expectations; I basically made this up.

I could see `?course-code=` also being fine and not relying on case sensitivity. I wouldn't recommend `?course_code` because underscores in URLs sometimes do not appear nicely if the page is printed out for example.